src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TakeBlockOnActivationSystem.java

```diff
@@ -63,14 +60,15 @@ public void onActivate(ActivateEvent event, EntityRef entity) {
         inventoryManager = new InventoryAuthoritySystem();
 
         BlockComponent blockComponent = entity.getComponent(BlockComponent.class);
+        LASTeam teamComponent = entity.getComponent(LASTeam.class);
 
         EntityRef flagTaker = event.getInstigator();
-        if (blockComponent.getBlock().getBlockFamily().getURI().toString().equals("LightAndShadowResources:blackFlag")) {
-            inventoryManager.giveItem(flagTaker, EntityRef.NULL, blockFactory.newInstance(blockManager.getBlockFamily("LightAndShadowResources:blackFlag")));
-        }
-        if (blockComponent.getBlock().getBlockFamily().getURI().toString().equals("LightAndShadowResources:redFlag")) {
+        if (teamComponent.team.equals("red")){
```

Comment:
missing space between `)` and `{`
Please do a code cleanup (Ctrl + Alt + L on IntelliJ)
You can also reformat code on a whole directory. Open up the project view with (Alt + 1) and right click a folder + click "Reformat Code"